docs/starters.yml

```diff
@@ -1634,3 +1634,20 @@
     - GSAP ready
     - Nice devtools
     - Decorators
+- url: https://gatsby-sseon-starter.netlify.com/
+  repo: https://github.com/SeonHyungJo/gatsby-sseon-starter
+  description: Simple starter template for Gatsby
+  tags:
+    - Simple
+    - Blog
+    - Disqus
+    - Portfolio
+    - Markdown
+    - Sseon
+  features:
+    - SASS/SCSS
+    - Comes with React Helmet for adding site meta tags
+    - Add Disqus
+    - Nice Pagination
+    - Prettier, ESLint & Stylelint
```

Comment:
I haven't found any indication of eslint or stylelint. Could you add it to your starter or remove it from the features list.